packages/dynamic-links/android/src/main/java/io/invertase/firebase/dynamiclinks/ReactNativeFirebaseDynamicLinksModule.java

```diff
@@ -435,7 +441,8 @@ public void onNewIntent(Intent intent) {
                               "dynamic_links_link_received",
                               dynamicLinkToWritableMap(
                                   pendingDynamicLinkData.getLink().toString(),
-                                  pendingDynamicLinkData.getMinimumAppVersion())));
+                                  pendingDynamicLinkData.getMinimumAppVersion(),
+                                  Arguments.makeNativeMap(pendingDynamicLinkData.getUtmParameters())));
```

Comment:
I can count parentheses, promise :crossed_fingers: 
```suggestion
                                  Arguments.makeNativeMap(pendingDynamicLinkData.getUtmParameters()))));
```